handler/handler.go

```diff
@@ -83,6 +83,9 @@ func GetTaskInfo(scheduler types.Scheduler) http.HandlerFunc {
 		id := vars["taskId"]
 		logrus.WithField("task_id", id).Debug("Fetching task")
 		task, _ := database.ReadTask(id)
```

Comment:
It might be good to provide a ReadMaskedTask or similiar to avoid accidentally exposing the masked fields in some other read operation.